### PR TITLE
4.14 GA

### PR DESCRIPTION
**Pull request by kalexand-rh**
Similar to https://github.com/openshift/openshift-docs/pull/59962, the 4.13 GA PR.

@adellape @gabriel-rh, will you PTAL?

### CODE REVIEW

File: .s2i/httpd-cfg/01-commercial.conf [hunk 1]
```diff
@@ -153,7 +153,7 @@ AddType text/vtt                            vtt
 
     # Redirects for "latest" version
     RewriteRule ^(container-platform|enterprise)/?$ /container-platform/latest [R=301]
-    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.13/$2 [NE,R=301]
+    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.14/$2 [NE,R=301]
```
Comment (kalexand-rh) [inline .s2i/httpd-cfg/01-commercial.conf:156]: @bscott-rh, please confirm these redirects.

File: _distro_map.yml [hunk 2]
```diff
@@ -156,7 +162,7 @@ openshift-dedicated:
     enterprise-3.11:
       name: '3'
       dir: dedicated/3
-    enterprise-4.13:
+    enterprise-4.14:
```
Comment (kalexand-rh) [inline _distro_map.yml:165]: @karanthshashank, please confirm that ROSA and OSD should start tracking 4.14 at OCP 4.14 GA.

File: .s2i/httpd-cfg/01-commercial.conf (hunk 1 above)
Comment (bscott-rh) [reply .s2i/httpd-cfg/01-commercial.conf:156]: Redirects LGTM

**Comment by kalexand-rh** [comment]
@aireilly, will you PTAL at this PR and confirm that this is treating the builds correctly?

File: _distro_map.yml (hunk 2 above)
Comment (karanthshashank) [reply _distro_map.yml:165]: Yes ACK, we will merge to 4.14.

File: _templates/_page_openshift.html.erb [hunk 3]
```diff
@@ -236,6 +237,7 @@
             </a>
             <select id="version-selector" onchange="versionSelector(this);">
               <option value="latest">latest</option>
+              <option value="4.14">4.13</option>
```
Comment (adellape) [inline _templates/_page_openshift.html.erb:240]:
```suggestion
              <option value="4.14">4.14</option>
```

File: index-community.html [hunk 4]
```diff
@@ -50,7 +50,8 @@ <h1 class="cover-heading">Documentation</h1>
              <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Select Version<span class="caret"></span></button>
              <ul class="dropdown-menu" role="menu">
                <li><a href="latest/"><i class="fa fa-arrow-circle-o-right"></i> OKD Latest</a></li>
-               <li><a href="4.13/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.13</a></li>
+               <li><a href="4.13/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.14</a></li>
+               <li><a href="4.14/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.13</a></li>
```
Comment (adellape) [inline index-community.html:54]: Are these backwards?
```suggestion
               <li><a href="4.14/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.14</a></li>
               <li><a href="4.13/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.13</a></li>
```

**Review by adellape — APPROVED** [review]

**Comment by kalexand-rh** [comment]
The last OCP 4.14.0 just shipped live. Merging.

**Pull request merged**